python/setup.py

```diff
@@ -45,11 +58,11 @@ def symlink_keyvi():
     # workaround for autowrap bug (includes incompatible boost)
     autowrap_data_dir = "autowrap_includes"
 
-    dictionary_sources = path.abspath('keyvi')
-    tpie_build_dir = path.join(dictionary_sources, '3rdparty/tpie/build')
-    tpie_install_prefix = 'install'
-    tpie_include_dir = path.join(tpie_build_dir, tpie_install_prefix, 'include')
-    tpie_lib_dir = path.join(tpie_build_dir, tpie_install_prefix, 'lib')
+    dictionary_sources = path.abspath(keyvi_cpp_link)
+    keyvi_build_dir = path.join(keyvi_cpp, 'build')
```

Comment:
if I understood correctly this is where #36 should be implemented

Reply:
Yes, will redo #38 once this PR is merged.